pkg/pillar/cmd/zedmanager/handledomainmgr.go

```diff
@@ -46,7 +47,12 @@ func MaybeAddDomainConfig(ctx *zedmanagerContext,
 	if ns != nil {
 		AppNum = ns.AppNum
 	}
-	effectiveActivate := effectiveActivateCurrentProfile(aiConfig, ctx.currentProfile)
+
+	isDNiDnode := false
+	if aiConfig.DesignatedNodeID != uuid.Nil && aiConfig.DesignatedNodeID == ctx.nodeUUID {
+		isDNiDnode = true
```

Comment:
Should this be set to true on a single node cluster? For a non-kubevirt case?
It would make sense to add some comment about the expectations in those cases.